tests/unit/lms/services/vitalsource/client_test.py

```diff
@@ -68,9 +71,67 @@ def test_it_signs_lti_launch_form_params(self, svc, lti_user):
             "oauth_version": "1.0",
         }
 
+    def test_api_get(self, svc, http_service):
+        svc._api_get("endpoint/path")  # pylint: disable=protected-access
+
+        http_service.get.assert_called_once_with(
+            "https://api.vitalsource.com/v4/endpoint/path",
+            headers={"X-VitalSource-API-Key": "api_key"},
+        )
+
+    def test_book_info_api(self, svc, book_info_schema):
+        with mock.patch.object(VitalSourceService, "_api_get") as api_get:
```

Comment:
If `_api_get` were moved into a separate helper class then you could patch it with a patch fixture which I think is a bit nicer than using `patch.object()`. Not a big deal though